front/src/components/auth_form/RegForm.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Button, Form} from "semantic-ui-react";
+import {useFormik} from "formik";
+import * as Yup from "yup";
+
+const RegForm = () => {
```

Comment:
Registration form and login are almost the same, you can make it as one component and render name field conditionally